memberOrganizationsBenefits.sh

```diff
@@ -4,7 +4,7 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-
+set -x
```

Comment:
I assume this was only required for debugging?

Reply:
good catch!